### DISCUSSION TITLE
Sample data for 1st gen Mazdas?

### DISCUSSION THREAD

**Discussion by Spiffkey** [MatthewKuKanich/CAN_Commander#5] [Q&A]
Hey @MatthewKuKanich, I found this repository because I was looking for data on the CAN communication codes for my 2005 Mazda 3. Since the LCD in my car works over CAN I want to be able to control it via the CAN bus to display whatever I want. I would also want to be able to read sensor data and display it on the LCD because if I'm already in the CAN system, why not. When I was reading the forum posts and watching the YouTube videos that brought me here, I saw that you had a 2007 Mazda 6. Since our cars are the same make and generation, I would be interested in seeing if the codes from your car are the same (or at least similar) as mine. I'm sure you're familiar with this website already because it's one of the first things that shows up when looking for Mazda CAN codes: [Mazda CAN Bus](https://www.madox.net/blog/projects/mazda-can-bus/) (madox.net). But there isn't very much detailed information and the table isn't filled in all the way so there isn't a ton of information. Not to mention, the post is ~15 years old so I wouldn't expect him to remember much. I was wondering if you could share the data you found (which IDs and bits correspond to which sensors) with me to make my life easier. And if it's already posted somewhere, please point me in that direction. I would double-check it with the data from madox.net and with some of my own sniffing, but I think it would be helpful for me. In any case, thanks for the repository. Whether or not I can use your data, this repository is exactly what I need to get my project working the way I want it to.

**Comment by jhahvhi** [comment]
Hi. I'm waiting like you. I got a dashboard from a first-generation Mazda 6 (it hasn't arrived yet) and I plan to use it on my computer in a simulator. I've seen some very interesting stuff, and on his YouTube channel he has videos showing how he practically decoded it and reads it with a Flipper Zero. But I haven't found the program anywhere. On the Mazda6Forum, he posted a partially decoded section before creating the program for Flipper Zero. I've asked him on various platforms to please upload it, either the program or a file with the commands and what each one is, but I haven't received a response. Have you made any progress?

**Comment by NorthBaandee** [comment]
I have almost finished the f0 javascript but I have been robbed of time for
this project for a few months while i lookafter my wife and refurbish a
caravan. I will try to upload some stuff next weekend if you remind me 8*)
